src/commands/ProjectCommand.ts

```diff
@@ -0,0 +1,60 @@
+import Command from "../abstracts/Command";
+import {Message, MessageEmbed} from "discord.js";
+import {readFile} from "fs";
```

Comment:
Why read the file rather than just import it? You could just do:
```ts
import projects from "../../assets/projects.json";
```

Doing so would also reduce a lot of logic you have within the class.

Reply:
Agreed (and good point, I didn't know imports like this worked in javascripty stuff). On contrary, are you sure you want this perma loaded in memory?

Reply:
Yeah, should be fine loaded.

Reply:
Resolved by exposing a (rather ugly) function which provides the project data, which in turn is stubbed by the tests. if you know a better way to stub the import lmk